src/status_im/contexts/preview/quo/wallet/transaction_progress.cljs

```diff
@@ -142,8 +142,8 @@
                 :epoch-number-mainnet         "181,329"
                 :epoch-number-optimism        "181,329"
                 :epoch-number-arbitrum        "181,329"
-                :optimism-progress-percentage "10"
```

Comment:
curious, is this a result of adding the schema? :)

Reply:
Yes, some use cases were wrong and I fixed them